### ISSUE TITLE
HDMI Not working for 2nd gen

### ISSUE THREAD

**Issue by san40511** [influenist/Mi-NB-Gaming-Laptop-MacOS#10]
Hello. I'm using your build and intalled Mojave but hdmi and sound drivers is not working on my xiaomi gaming laptop. Could you help me with my problem. Please tell me what should i do with it? thanks

**Comment by fanqieVip** [comment]
Are you sure !  It‘s working fine on my xiaomi gaming laptop. Please check your operation !

**Comment by san40511** [comment]
yes. i'm sure. only  type c works fine after gamecard patching if run without patch then system reloaded

**Comment by san40511** [comment]
I decided reintall the OS. I will report here about results. Maybe i did something wrong

**Comment by san40511** [comment]
this time i even can't run recovery to reinstall osx.

**Comment by influenist** [comment]
Hi @san40511 thanks for reaching out to us here. Please load up the EFI from the attachment on an USB drive and let me know. I had a some strange issue during the usage of Mojave myself last month (without an update) and needed to adjust the EFI. Please note I need to make some final adjustment, so it has not pushed to the repo yet as I'm working on a MacPro 2013 replica a.t.m. which I will publish later also on Github. If this EFI doesn't work, please check out the repo which is designed for the gaming laptop by @johnnync13 https://github.com/johnnync13/XiaomiGaming

* Please note in this build webcam USB is not working, yet USB-C to HDMI is. I tested it with this hub: https://www.aliexpress.com/item/33044856328.html?spm=a2g0s.9042311.0.0.74294c4dckyKWA HDMI output works also.

[EFI.zip](https://github.com/influenist/Mi-NB-Gaming-Laptop-MacOS/files/3532180/EFI.zip)

**Comment by san40511** [comment]
i took this build https://github.com/daliansky/XiaoMi-Pro-Hackintosh + Apple typec adapter like this https://static.bhphoto.com/images/images2500x2500/1426074342_1128930.jpg. this adapter is fully working (usb, hdmi and type c). this build works perfect for me. System: Mojave:10.14.3 , not working devices : bluetooth (use csr 4.0 - setup took about 3 days but now it works perfect), wifi (used netis adapter) and microphone(not fixed yet - alc profiles not fixed the issue).
